tests/test_client_check.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import os
+import sys
+import pytest
+import tempfile
+import subprocess
+import configparser
+from types import SimpleNamespace
+from urlchecker.client.check import main
+
+
+def test_installation():
```

Comment:
I've never seen a test for installation, is it necessary? Since we are installing it first in the CI, wouldn't that catch any issues that might arise? I'm worried that having this (and a user testing it locally before install, and using as a local module) would not expect something to install with pip3. I think it would be best to remove this test, and have the install tested by way of install in the github actions.

Reply:
okay, I will remove this.